lib/scala/editions/src/test/scala/org/enso/editions/EditionResolverSpec.scala

```diff
@@ -48,6 +48,7 @@ class EditionResolverSpec
         libraries     = Map()
       )
     )
+    def editions = _editions
```

Comment:
seems unnecessary

Reply:
Not if I want to override *and* user the super value.